VulnerableApplication/Global.asax.cs

```diff
@@ -10,12 +10,18 @@ namespace VulnerableApplication
 {
     public class MvcApplication : System.Web.HttpApplication
     {
+        public string Password = "abcdef";
+    
```

Comment:
[CODE_SMELL] 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.